pacman/model/partitioner_interfaces/abstract_splitter_partitioner.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020-2021 The University of Manchester
```

Comment:
2021? _Slightly_ ahead of ourselves, but I guess it means that we shouldn't accept this PR until after new year? 😄